frontend/rolecall/src/app/services/super-ballet-display.service.ts

```diff
@@ -0,0 +1,27 @@
+import {Injectable} from '@angular/core';
+
+
+@Injectable({providedIn: 'root'})
+export class SuperBalletDisplayService {
+  superBalletIsOpenMap = new Map<string, boolean>();
+
+  changeOpenState(uuid: string, isOpen: boolean) {
```

Comment:
I would prefer to avoid using the word `uuid` in the UI, calling it `balletId` is better semantically as it gives more context to what it's identifying.

Reply:
Done.